source/guide_symfony.rst

```diff
@@ -0,0 +1,198 @@
+      :align: center
+
+##########
+Symfony
+##########
+
+.. tag_list::
+
+`Symfony <https://symfony.com/what-is-symfony>`_ is a set of PHP Components, a Web Application framework, a Philosophy, and a Community — all working together in harmony.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`domains <web-domains>`
+  * :manual:`PHP <lang-php>`
+
+License
+=======
+
+Symfony Source Code is published under the terms of the MIT License
+
+  * `<https://symfony.com/license>`_
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 8:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '8.0'
+ [isabell@stardust ~]$
+
+Your symfony URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Install symfony cli
+-------------------
+
+::
+
+ [isabell@stardust ~]$ wget https://get.symfony.com/cli/installer -O - | bash
+
+Modify ``~/.bash_profile`` to use symfony directly
+
+.. code-block:: ini
+
+ ...
+ # User specific environment and startup programs
+
+ PATH=$HOME/.local/bin:$HOME/bin:$PATH
+ PATH="$HOME/.symfony/bin:$PATH"
+ export PATH
+
+
+Create symfony project
+----------------------
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/isabell/
+ [isabell@stardust isabell]$ symfony new my_project
+ * Creating a new Symfony project with Composer
+   (running /usr/local/bin/composer create-project symfony/skeleton /var/www/virtual/isabell/my_project  --no-interaction)
+ * Setting up the project under Git version control
+   (running git init /var/www/virtual/isabell/my_project)
+ [OK] Your project is now ready in /var/www/virtual/isabell/my_project
+
+Adding Rewrite Rules
+
+::
+
+ [isabell@stardust isabell]$ composer require symfony/apache-pack
+ -  WARNING  symfony/apache-pack (>=1.0): From github.com/symfony/recipes-contrib:master
+    The recipe for this package comes from the "contrib" repository, which is open to community contributions.
+    Review the recipe at https://github.com/symfony/recipes-contrib/tree/master/symfony/apache-pack/1.0
+
+    Do you want to execute this recipe?
+    [y] Yes
+    [n] No
+    [a] Yes for all packages, only for the current installation session
+    [p] Yes permanently, never ask again for this project
+    (defaults to n): y
+  - Configuring symfony/apache-pack (>=1.0): From github.com/symfony/recipes-contrib:master
+ Executing script cache:clear [OK]
+ Executing script assets:install public [OK]
+
+ Some files may have been created or updated to configure your new packages.
+ Please review, edit and commit them: these files are yours.
+
+ Nothing to unpack
+
+Configuration
+=============
+
+Configure the web server
+------------------------
+
+We connect our symfony public folder to our web server
+
+::
+
+ [isabell@stardust ~]$ ln -s /var/www/virtual/isabell/my_project/public /var/www/virtual/isabell/html/
```

Comment:
Use variables and skip the final `/` because the `html` directory is not supposed to exist.
```suggestion
 [isabell@stardust ~]$ ln -s /var/www/virtual/$USER/my_project/public /var/www/virtual/$USER/html
```